radicle-cli/tests/framework/mod.rs

```diff
@@ -174,6 +174,7 @@ impl TestFormula {
                 }
                 let result = Command::new(cmd.clone())
                     .env_clear()
+                    .env("LANG", "C")
```

Comment:
This is to run the tests with English explicitly.
In my case e.g. I get issues showing output in german